requirements.txt

```diff
@@ -9,7 +9,7 @@ wsgiref==0.1.2
 pep8==1.4.6 # rq.filter: ==1.4.6
 flake8==2.0 # rq.filter: ==2.0
 python-omgeo==1.7.2
-django-registration-redux==1.2
+django-registration-redux==1.3
```

Comment:
According to the changelog on requires.io (https://requires.io/github/OpenTreeMap/otm-core/requirements/?branch=requires-io-develop) this has a bundle of new features that can get some good use out of!
 - Adds a `resend_activation_mail` method
 - Distinguishes between account activation failure and attempted reactivation.
 - Adds support for Django 1.9

I definitely want to do some smoke testing before we upgrade though.